src/main/java/com/minecolonies/util/BlockUtils.java

```diff
@@ -179,7 +179,7 @@ else if(blockState.getBlock() instanceof BlockDaylightDetector)
         }
         else if (blockState.getBlock() instanceof BlockDoor)
         {
-            return ((BlockDoor) blockState.getBlock()).getItem();
+            //return ((BlockDoor) blockState.getBlock()).getItem();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)